### ISSUE TITLE
Extension Failure: Managed Dependencies Off

### ISSUE THREAD

**Issue by GavinOsborn** [GoogleCloudPlatform/cloud-code-vscode#452]
Issue Type: <b>Bug</b>
We script our dev tooling onto our workstations. This includes the install of minikube, skaffold, vscode and the cloud-code extension. As part of this we also configure `cloudcode.autoDependencies` to be `off`.

When I attempt to issue any command, after a while (10s?) I will see a VS Code popup dialog that says:

> Command 'Cloud Code: Report Extension Issue on Github' resulted in an error (command 'cloudcode.reportIssueOnGithub' not found)

(obviously the command name will change depending on what I'm trying to do)

Looking through the "Log (Window)" output in my terminal I can see the following entry:

> [2021-07-25 18:03:07.288] [renderer1] [warning] UNRESPONSIVE extension host, 'googlecloudtools.cloudcode' took 99% of 5049.199ms, saved PROFILE here: 'file:///c%3A/Users/Gav/AppData/Local/Temp/exthost-9e0383.cpuprofile' [{"id":"gc","total":52148,"percentage":1},{"id":"googlecloudtools.cloudcode","total":4991458,"percentage":99},{"id":"program","total":2936,"percentage":0},{"id":"self","total":2488,"percentage":0}]

Things I have tried:
 - If I enable autoDependencies then everything works fine.
 - The error still occurs if I uninstall all other extensions

*Context*
- Managed Dependencies: off
- Cloud SDK Version: 349.0.0
- Skaffold Version: v1.27.0
- Minikube Version: 1.21.0

Extension version: 1.13.1
VS Code version: Code 1.58.2 (c3f126316369cd610563c75b1b1725e0679adfb3, 2021-07-14T22:10:15.214Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor             (24 x 3800)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.91GB (21.12GB free)|
|Process Argv|--crash-reporter-id 55c520e5-6bec-427d-b0b0-30823cafde4f|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt700cf:30270857
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551:30311712
vspre833:30321513
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscus158:30321503
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlangct:30333562
binariesv615:30325510
vsccppwt:30329788
pythonvssor306cf:30341832
bridge0708:30335490

```

</details>

**Comment by j-windsor** [comment]
Hi @GavinOsborn! Sorry you are having this issue. Does your script also install Google Cloud SDK (gcloud) to the `PATH` as well or are you skipping that?

**Comment by GavinOsborn** [comment]
Hi @j-windsor , we don't, no.
We have no known reason for installing the sdk as our usage of this tool chain is for sandboxed local development only. No gcloud, no remote repositories, no reason to authenticate to a remote system. 

Is that problematic?
The extension used to prompt you to download it if needs be.

**Comment by j-windsor** [comment]
It is not required: I was just confirming the repro. I can repro and we will be releasing a fix soon. It looks like we introduced a bug in the last few releases where we inadvertently require Google Cloud SDK to be installed.

**Comment by GavinOsborn** [comment]
Thanks!

**Comment by GavinOsborn** [comment]
Hey @j-windsor, in the interim are we able to confirm when the regression was introduced?
eg: which version of the extension I can revert back to for now?

**Comment by j-windsor** [comment]
It looks like the regression was introduced in 1.13.0. You can revert to 1.12.0 for now. I'll keep you updated on this issue when you can return to the current version.

**Comment by sivakku** [comment]
We will be addressing this in the next release in August. Thank for the feedback.

**Comment by j-windsor** [comment]
This has been fixed in Cloud Code v1.14.0. Thanks for reporting!
